miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -66,6 +67,9 @@ abstract class MiniAppMessageBridge {
             this.userInfoBridgeDispatcher.init(bridgeExecutor, customPermissionCache, miniAppId)
         else this.userInfoBridgeDispatcher = object : UserInfoBridgeDispatcher() {}
 
+        if (this::chatMessageBridgeDispatcher.isInitialized)
+            this.chatMessageBridgeDispatcher.init(bridgeExecutor, customPermissionCache, miniAppId)
+
```

Comment:
must have this to avoid the null crash when miniapp is initialized with `MiniAppMessageBridge` first and user does not call `setChatMessageBridgeDispatcher`
```suggestion
else this.chatMessageBridgeDispatcher = object : ChatMessageBridgeDispatcher() {}
```

Reply:
I have updated this part. In this case, we need to implement `sendMessageToContact` with an empty function block for default initialization.